A45-retry-stats.md

```diff
@@ -0,0 +1,126 @@
+A55: Exposing OpenCensus Metrics and Tracing for gRPC retry
+----
+* Author(s): dapengzhang0, dfawley, ejona86, markdroth
+* Approver: markdroth
+* Status: In Review
+* Implemented in: C-core, Java
+* Last updated: 2021-08-02
+* Discussion at: https://groups.google.com/g/grpc-io/c/EAbFqaJWp5w
+
+## Abstract
+
+This document is a proposal on how gRPC client would expose OpenCensus metrics and tracing when retry is enbaled. It can be considered as an augment of the [retry gRFC](https://github.com/grpc/proposal/blob/master/A6-client-retries.md).
```

Comment:
Just use the file name as the link. Ditto elsewhere for gRFC references.

I think I actually need to do the same in my recent gRFC. I only noticed because I saw it recently on another gRFC (Sanjay's?).

Reply:
Done.